frontend/src/app/messages/messages.component.html

```diff
@@ -0,0 +1,8 @@
+<p *ngIf="showMessages">
+  <ngb-alert
+    *ngFor="let message of messages$ | async"
+    [type]="message.type ==='error' ? 'warning': message.type"
+    [dismissible]="false">
```

Comment:
I would make it dismissable and give it a longer timout time, see my comment in component.ts file.